package.json

```diff
@@ -15,12 +15,13 @@
     "test:proposals": "forge test --match-path './test/proposals/**' --fork-url $ETH_RPC_URL -vvv",
     "test:integration": "forge test --match-path './test/integration/**' --fork-url $ETH_RPC_URL -vvv",
     "coverage": "forge coverage --fork-url $ETH_RPC_URL --match-path './test/**' --report lcov --report summary",
-    "coverage:unit": "forge coverage --match-path './test/unit/**' --report lcov --report summary",
+    "coverage:unit": "forge coverage --match-path './test/unit/**' --report summary --report lcov",
```

Comment:
On my machine, the other way around creates the .lcov but not the cli report

Using this order (summary then lcov) works